src/main/java/mekanism/common/item/gear/ItemJetpack.java

```diff
@@ -87,6 +87,11 @@ public JetpackMode getJetpackMode(ItemStack stack) {
         return getMode(stack);
     }
 
+    @Override
+    public double getJetpackThrust(ItemStack stack) {
+        return 0.1D;
```

Comment:
I believe this used to be `0.15`, is the reason for this change just to balance it out a bit more with the vector mode or was it accidental?

Reply:
This was an intentional change. This didn't really have anything to do with the vector mode, it was more for balancing with the exponential fall off change that also affects normal mode. I'm open to changing it to whatever you want but now that the y velocity isn't capped this has more relevance. Previously changing the thrust only changed if you reached the max y velocity in 2 vs. 3 ticks. So not a big change even it was reduced a lot.